docs/conf/quotes.txt.example

```diff
@@ -49,7 +49,7 @@ Computers make very fast, very accurate mistakes
 Life would be so much easier if we only had the source code
 Who is this 'General Failure' and why is he reading my disk?
 hAS aNYONE sEEN MY cAPSLOCK kEY?
-InspIRCd, now with excessive ammounts of Cheeze
+InspIRCd, now with excessive amounts of Cheeze
```

Comment:
I'm leaving `Cheeze` -- happy to drop this, note that the spell-checker was told to ignore this file after the original list of typos was selected.

Reply:
thank you ;)